### PR TITLE
Update README.md to indicate that these scripts are no longer needed

### PR DESCRIPTION
**Pull request by dwnusbaum**
This PR should not be merged until the new release is out. Once it is merged, I will archive the repo from the GitHub UI to make the status clearer.

You can see a rendered version of this PR [here](https://github.com/cloudbees/user-activity-monitoring-scripts/blob/4f2c7c8923aefb8ebdec16f6a86801190d7c7ad5/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+:warning: **The scripts in this repository are no longer needed as of [User Activity Monitoring plugin version 1.4](https://docs.cloudbees.com/docs/release-notes/latest/plugins/user-activity-monitoring-plugin/1.4).**
```
Comment (dwnusbaum) [inline README.md]: TODO: Verify that this link works correctly once the release is out before merging the PR.

Comment (dwnusbaum) [reply README.md]: I had the wrong version initially, but https://docs.cloudbees.com/docs/release-notes/latest/plugins/user-activity-monitoring-plugin/1.3 does work correctly.

**Comment by jglick** [comment]
:information_source: A URL like https://github.com/dwnusbaum/user-activity-monitoring-scripts/blob/user-activity-monitoring-1.4/README.md will reflect any subsequent edits.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+:warning: **The scripts in this repository are no longer needed as of [User Activity Monitoring plugin version 1.3](https://docs.cloudbees.com/docs/release-notes/latest/plugins/user-activity-monitoring-plugin/1.3).**
```
Comment (jglick) [inline README.md:1]: Version number noted here (2×) does not match that in the PR branch; which is right?

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
(Also, for _edits_ rather than _new files_, the rich diff button in GH is typically more helpful.)

File: README.md (hunk 2 above)
Comment (dwnusbaum) [reply README.md:1]: 1.4 is the release that went out yesterday, but 1.3 is the version with the corresponding changes, so the release notes for 1.3 have the relevant message. When I initially filed the PR I forgot about the intermediate 1.3 release.

**Pull request merged**